examples/network_traffic_prediction/AirTrafficPrediction.ipynb

```diff
@@ -0,0 +1,735 @@
+     "output_type": "stream",
+     "text": [
+      "INFO:root:airport does not contains NoneWe will do inference on the entire node set.\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "========================== Do Inference ================================\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Setup dataloader for inference\n",
+    "print(f'========================== Do Inference ================================')\n",
+    "infer_dataloader = GSgnnNodeDataLoader(\n",
+    "    dataset=ml_data,\n",
+    "    target_idx=ml_data.get_node_infer_set(ntypes='airport', mask=None),\n",
+    "    node_feats=nfeats_4_modeling,\n",
+    "    edge_feats=efeats_4_modeling,\n",
+    "    label_field='inventory_amounts',\n",
+    "    fanout=fanout,\n",
+    "    batch_size=64,\n",
+    "    train_task=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4f271746-c51e-4138-8543-b69b8fbc34bd",
+   "metadata": {},
+   "source": [
+    "GraphStorm provides a set of `Inferrer`s that can perform highly efficient inference for very large graphs. GraphStorm's `GSgnnNodePredictionInferrer` cannot handle the sliding window mehtod either. Similarly we extends the `GSgnnNodePredictionInferrer`, and specifically implements the autoregressive sliding window method by replacing true time serie values with the predicted ones, and then call the same customized `node_mini_batch_gnn_predict()` method during inference. Users can find the details in the `nr_models.py` file."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 28,
+   "id": "270dc5b7-8cb0-4515-abc1-5c2bbca36836",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from nr_models import NodePredictionInferrer4TS\n",
+    "\n",
+    "# Run inference on the inference dataset\n",
+    "infer = NodePredictionInferrer4TS(model)\n",
+    "\n",
+    "infer.infer(infer_dataloader,\n",
+    "            use_mini_batch_infer=True,\n",
+    "            save_embed_path=None,\n",
+    "            init_step=0,\n",
+    "            infer_steps=7,\n",
+    "            node_id_mapping_file='./gs_1p/node_mapping.pt',\n",
+    "            save_prediction_path='./predictions')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f252cfb9-ac19-463d-86e0-ba24347ab123",
+   "metadata": {},
+   "source": [
+    "Now let's check the prediction results"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 29,
+   "id": "90af92cd-dfcb-4ae7-b29d-fe536e1f6ed3",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "total 24K\n",
+      "-rw-rw-r-- 1 ubuntu ubuntu  15K Nov 21 00:06 predict-00000.pt\n",
+      "-rw-rw-r-- 1 ubuntu ubuntu 4.9K Nov 21 00:06 predict_nids-00000.pt\n"
+     ]
+    }
+   ],
+   "source": [
+    "# The GNN embeddings and predictions on the inference graph are saved to the folder named after the target_ntype\n",
```

Comment:
Do we generate GNN embeddings in this example?

Reply:
No. Removed.